cmd/noms/diff/diff_test.go

```diff
@@ -70,7 +72,32 @@ func createStruct(name string, kv ...interface{}) types.Struct {
 	return types.NewStruct(name, fields)
 }
 
-func TestNomsMapdiff(t *testing.T) {
+func pathsFromDiff(v1, v2 types.Value, leftRight bool) []string {
+	dChan := make(chan Difference)
+	sChan := make(chan struct{})
+
+	go func() {
+		Diff(v1, v2, dChan, sChan, leftRight)
+		close(dChan)
+	}()
+
+	paths := []string{}
```

Comment:
You should probably declare this like `var paths []string` so that it's nil when there are no paths. Or even name the return value.